File: structures/linearSearch/linearSearch.js
// Big O: O(n)
const list = [12, 34, 56, 78, 90];

function linearSearch(list, item) {
  for (let index = 0; index < list.length; index++) {
    if (list[index] === item) {
      return index;
    }
  }

  return null;
}

console.time('linearSearch');
console.log(linearSearch(list, 90));
console.timeEnd('linearSearch');

File: structures/linearSearch/linearSearch.py
#  Big O: O(n)
list = [12, 34, 56, 78, 90]


def linearSearch(list, item):
    """Returns index if element is found, None otherwise"""
    for index in range(len(list)):
        if list[index] == item:
            return index
    return None


print(linearSearch(list, 56))
